spanner/api/Spanner.Samples.Tests/CopyBackupWithMRCMEKTest.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright 2022 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.Spanner.Admin.Database.V1;
+using System;
+using Xunit;
+
+[Collection(nameof(SpannerFixture))]
+public class CopyBackupWithMRCMEKTest
+{
+    private readonly SpannerFixture _spannerFixture;
+
+    public CopyBackupWithMRCMEKTest(SpannerFixture spannerFixture)
+    {
+        _spannerFixture = spannerFixture;
+    }
+
+    [Fact]
+    public void CopyBackupWithMRCMEK()
+    {
+        CopyBackupWithMRCMEKSample copyBackupWithMRCMEKSample = new CopyBackupWithMRCMEKSample();
```

Comment:
You cannot use the same backup IDs that are being used for the other backup tests. You need specific IDs for these tests, and make certain the backups are there when you need them, etc.